src/index.html

```diff
@@ -11,7 +11,180 @@
+            id="birthdate"
+          >
+        </label>
+
+        <label class="form_field" for="term">
+          I accept the term of the agreement:
+          <input
+            type="checkbox"
+            name="term"
+            id="term"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form_fieldset">
+        <legend>Registration</legend>
+
+        <label class="form_field" for="email">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form_field" for="password">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="6"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form_fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form_field">
+          <label >Do you love cats?</label>
+          <label for="cats_yes">
+            <input
+              type="radio"
+              name="cats"
+              id="cats_yes"
+            >
+            Yes
+          </label>
+          <label for="cats_no">
+            <input
+              type="radio"
+              name="cats"
+              id="cats_no"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form_field" for="color">
+          What is your favorite color?
+          <input
+            type="color"
+            name="color"
+            id="color"
+          >
+        </label>
+
+        <label class="form_field" for="timeToBed">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="timeToBed"
+            id="timeToBed"
+            step="1"
+          >
+        </label>
+
+        <label class="form_field" for="cars">
+          What are your favourite brand of cars?
+          <select
+            name="cars"
+            id="cars"
+            multiple
+          >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form_field" for="rate">
+          How do you rate our work?
+          <input
+            type="range"
+            name="rate"
+            id="rate"
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form_fieldset">
+        <legend>Additional info:</legend>
+
+        <label class="form_field" for="comments">
+          Comments:
+          <textarea name="comments" id="comments" ></textarea>
+        </label>
+
+        <label class="form_field" for="recommend">
+          Would you recommend us?
+          <select
+          name="recommend"
+          id="recommend"
+        >
```

Comment:
3 point in checklist
```suggestion
          <select
            name="recommend"
            id="recommend"
        >
```